<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>log</class>
 <widget class="QWidget" name="log">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查看日志</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>static/ico.png</normaloff>static/ico.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>日志</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="log_text">
     <property name="plainText">
      <string>当前没有日志！</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="close_button">
       <property name="text">
        <string>我知道了</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear_log">
       <property name="text">
        <string>清空日志</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
